src/contribute/release-guide.md

```diff
@@ -255,32 +268,18 @@ Copy the source release to the dev repository of `dist.apache.org`.
 
 1. Verify that files are [present](https://dist.apache.org/repos/dist/dev/beam).
 
-### Build the API reference
-
-Beam publishes API reference manual for each release on the website. For Java SDK, that’s Javadoc.
-
-Check out release candidate, as follows:
-
-    git checkout ${TAG}
-
-Use Maven Javadoc plugin to generate the new Java reference manual, as follows:
-
-    mvn -DskipTests clean package javadoc:aggregate \
-        -Ddoctitle="Apache Beam SDK for Java, version ${VERSION}" \
-        -Dwindowtitle="Apache Beam SDK for Java, version ${VERSION}" \
-        -Dmaven.javadoc.failOnError=false \
-        -DexcludePackageNames="org.apache.beam.examples,org.apache.beam.runners.dataflow.internal,org.apache.beam.runners.flink.examples,org.apache.beam.runners.flink.translation,org.apache.beam.runners.spark.examples,org.apache.beam.runners.spark.translation,org.apache.beam.runners.apex.translation,org.apache.beam.sdk.microbenchmarks.coders.generated,org.apache.beam.sdk.microbenchmarks.transforms.generated,org.openjdk.jmh.infra.generated"
-
-By default, the Javadoc will be generated in `target/site/apidocs/`. Let `${JAVADOC_ROOT}` be the absolute path to `apidocs`. ([Pull request #1015](https://github.com/apache/beam/pull/1015) will hopefully simplify this process.)
-
-Please carefully review the generated Javadoc. Check for completeness and presence of all relevant packages and `package-info.java`; consider adding less relevant packages to the `excludePackageNames` configuration. The index page is generated at `${JAVADOC_ROOT}/index.html`.
-
 ### Propose a pull request for website updates
 
 The final step of building the candidate is to propose a website pull request.
 
 Start by updating `release_latest` version flag in the top-level `_config.yml`, and list the new release in the [Apache Beam Downloads]({{ site.baseurl }}/get-started/downloads/), linking to the source code download and the Release Notes in JIRA.
 
+Beam publishes API reference manual for each release on the website. For Java SDK, that’s Javadoc.
+
+One of the artifacts created in the release contains the Javadoc for the website: 
+`org.apache.beam:beam-sdks-java-javadoc`. To update the website, you must unpack
+this jar file from the release candidate into the source tree of the website.
+
```

Comment:
can this all be moved into one bullet below?

* Unpack the X jar built during the release to some temp location.
* Copy ...